BackendApartmentReservation/BackendApartmentReservation/Entities/Travel.cs

```diff
@@ -0,0 +1,18 @@
+﻿using BackendApartmentReservation.Utilities;
+using System;
+
+namespace BackendApartmentReservation.Entities
+{
+    public class Travel
+    {
+        public int Id { get; set; }
+        public double Cost { get; set; }
+        // public Office DestinationOffice { get; set; }
```

Comment:
Lets not commit commented out code:D or change it to TODO

Reply:
Will delete the comment